lib/spaceship/portal/device.rb

```diff
@@ -121,7 +121,7 @@ def find(device_id, mac: false)
         # @return (Device) Find a device based on the UDID of the device. nil if no device was found.
         def find_by_udid(device_udid, mac: false)
           all(mac: mac).find do |device|
-            device.udid == device_udid
+            device.udid.downcase == device_udid.downcase
```

Comment:
using `casecmp` returns `0` instead of `false` which leads some extra code to be written. this is nicer IMO. and `rubocop` doesn't mind it.